tests/integration/roles/test_create/defaults/main.yml

```diff
@@ -67,7 +67,7 @@ defaults:
             enable_bpdu_guard: true
           topology_switch_trunk_po_interface:
             description: 'NetAsCode Trunk PO Interface'
-            mtu: jumbo
+            mtu: jumbo            
```

Comment:
Get rid of what looks like trailing whitespace

Reply:
will do.  This was an exact copy of the current defaults main.yaml.  I'll look at both files and remove any extra spaces.

Reply:
I fixed the trailing whitespace in the default main.yml in the validate role